sanssouci/post_hoc_bounds.py

```diff
@@ -39,11 +39,49 @@ def max_fp(p_values, thresholds):
         Annals of Statistics, 48(3), 1281-1303.
     """
 
+    s = len(p_values)
+    if s == 0:
+        return(0)
```

Comment:
```suggestion
        return 0
```
Is this line covered by a unit test ?